python/cucim/src/cucim/skimage/exposure/tests/test_exposure.py

```diff
@@ -14,6 +15,10 @@
 from cucim.skimage.exposure.exposure import intensity_range
 from cucim.skimage.util.dtype import dtype_range
 
+# TODO: Some tests fail unexpectedly on ARM.
+ON_AARCH64 = platform.machine() == "aarch64"
+ON_AARCH64_REASON = "TODO: Test fails unexpectedly on ARM."
```

Comment:
We decided to skip these after discussing with @grlee77 offline. Noting this here for clarity. Also for reference in a follow up issue